UPGRADE-3.2.md

```diff
@@ -27,6 +27,7 @@ FrameworkBundle
  * The `Resources/public/images/*` files have been removed.
  * The `Resources/public/css/*.css` files have been removed (they are now inlined
    in TwigBundle).
+ * The `Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler\AddConsoleCommandPass` has been deprecated. Use `Symfony\Component\Console\DependencyInjection\AddConsoleCommandPass` instead.
```

Comment:
Ah sorry, now that Symfony 3.2 is released we would need to move this to the `UPGRADE-3.2md` file and update deprecation messages accordingly.